2_observations.yml

```diff
@@ -134,3 +135,20 @@ targets:
       mgd_to_cms = 0.0438, cfs_to_cms = 0.0283,
       mg_to_cm = 4404.88, ft_to_m = 0.3048)
 
+ # reservoir surface area data
+ # you must download reservoir Monthly Shapefiles from here: 'http://umnlcc.cs.umn.edu/realsat/'
+ # unzip downloaded folder to 2_observations/in
+ # such that the structure becomes
+ # 2_observations/in/ID-Number
+      # in/573567/573567
```

Comment:
Let's add another layer of nesting here, e.g., `2_observations/in/realsat/573567`, so that these folder names are more clearly distinguished from others that may exist or get added later to `2_observations/in`.